src/server/status/index.js

```diff
@@ -21,6 +21,8 @@ import ServerStatus from './server_status';
 import { Metrics } from './lib/metrics';
 import { registerStatusPage, registerStatusApi, registerStatsApi } from './routes';
 import { getOpsStatsCollector } from './collectors';
+import Oppsy from 'oppsy';
```

Comment:
This needs to be added as a dependency if we are directly using it.

Reply:
Ah good call.